.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: Release to npm
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version number to tag and release'
+        required: true
+        type: string
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    environment: release
+    permissions:
+      contents: write
+      id-token: write
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm install -g npm@latest
+      - run: npm install
+      - name: Change version number and push
+        run: |
+          git config --global user.email "hello@matteocollina.com"
+          git config --global user.name "Matteo Collina"
+          npm version ${{ inputs.version }} --git-tag-version=false
```

Comment:
we need to update the fastify.js > version const too, or the release will fail

https://github.com/fastify/fastify/blob/c90ac14c7179606bb0833447dda1b83007156fa5/fastify.js#L3

Reply:
Ugh, wish we didn't do that.

Reply:
Can we not just import this from package i.e `const { version: VERSION } = require("../package.json");`?

Reply:
That's what we originally did. People that insist on mangling the source code they wrote into an unusable mess found that their tools don't like that.

Reply:
https://github.com/pinojs/pino/blob/master/inc-version.sh

Reply:
Updated, this should be resolved.